test/src/test/java/hudson/util/ProcessTreeKillerTest.java

```diff
@@ -85,7 +112,6 @@ public void processProperlyKilledUnix() throws Exception {
 
     @Test
     public void doNotKillProcessWithCookie() throws Exception {
-        Assume.assumeFalse("This test does not involve windows", Functions.isWindows());
```

Comment:
Why was this bit changed there is a comment in #2351 about this

Reply:
8230f645a445ab4f51ae7a77b9c1a6c0b29b790c commit message says why: because I fixed this test and now it passes on Windows.

Reply:
Sorry about that